android/src/main/java/com/voximplant/reactnative/VICallModule.java

```diff
@@ -62,6 +67,7 @@ public void internalSetup(String callId) {
         ICall call = CallManager.getInstance().getCallById(callId);
         if (call != null) {
             call.addCallListener(this);
+            call.setQualityIssueListener(this);
```

Comment:
You need to unsubscribe from quality issue events when call is disconnected or failed